properties/P000069.md

```diff
@@ -3,8 +3,20 @@ uid: P000069
 name: Strategic Menger
 aliases:
   - M^+
+  - Strategically Menger
+  - \Omega M^+
+  - Strategically $\omega$-Menger
```

Comment:
Throughout: both `Omega` and `omega` return the same search results, so choose the canonical one for your alias.

Reply:
Check